interactors/text-field.js

```diff
@@ -0,0 +1,35 @@
+import { createInteractor, perform, TextField } from '@bigtest/interactor';
+
+import IconButton from './icon-button';
+
+const label = (el) => {
+  const labelText = el.querySelector('label');
+  return labelText ? labelText.innerText : undefined;
+};
+
+export default createInteractor('text field')({
+  selector: 'div[class^=textField-]',
+  locator: label,
+  filters: {
+    id: (el) => el.querySelector('input').id,
+    label,
+    type: (el) => el.querySelector('input').type,
+    value: (el) => el.querySelector('input').value,
+    focused: (el) => el.querySelector('input').contains(el.ownerDocument.activeElement),
+    readOnly: (el) => el.querySelector('input').hasAttribute('readOnly'),
```

Comment:
/cc @elrickvm, maybe this is better than the double bang we were doing last night.

Reply:
nice!